src/components/SearchForm.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <form class="search-form">
+    <input type="text" placeholder="Search" v-model="searchInput">
+    <article class="secondary-inputs">
+    <select name="genres" v-model="genreInput">
+      <option>Genres:</option>
+      <option>Comedy</option>
+      <option>Horror</option>
+    </select>
+    <select name="languages" v-model="languageInput">
+      <option>En</option>
+      <option>Esp</option>
+    </select>
+    <select name="regions" v-model="regionInput">
+      <option>US</option>
+      <option>CA</option>
+      <option>Belarus</option>
+    </select>
+    </article>
+    <button type="submit" v-on:click="searchMovies">Submit Search</button>
+  </form>
+</template>
+
+<script>
+
+export default {
+  name: 'SearchForm',
+  data() {
```

Comment:
We will need to add some props to this so we can inlcude the dynamic search categories.